internal/datastore/applause.go

```diff
@@ -0,0 +1,79 @@
+package datastore
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+type applause struct {
+	c          *config
+	activeUser map[int]bool
```

Comment:
plase name it `presentUsers` since `active` is a different state for a user.

Reply:
done